### PR TITLE
Accessibility fixes for Live region announcements and textfield error message

### PR DESCRIPTION
**Pull request by mdahamiwal**
#### Pull request checklist

- [x] Addresses an existing issue: #1272, #792, #1240 
- [x] Include a change request file if publishing 
- [x] bugfixes, accessibility enhancements
  - [x] Includes tests
- [] Documentation update

#### Description of changes
1. Extract a common live region control to create a live region element and update text with a delay. Will be used in MessageBar and textField for now.
2. TextField now announces error message properly.
3. glyph icon for error message to distinguish error message from label in high contrast mode.
image

For testing use following AT + browser combination: 
NVDA + Firefox 
or IE11/Edge + narrator, JAWS

### CODE REVIEW

File: packages/office-ui-fabric-react/src/common/DelayRender.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  public componentWillUnmount(): void {
+    if (this.renderTimeOut) {
+      clearTimeout(this.renderTimeOut);
+    }
+  }
+
+  public render(): JSX.Element {
+    const nativeProps = getNativeProps(this.props, divProperties);
+
+    return (
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/common/DelayRender.tsx]: there is a DelayedRender in utilities. is that not good enough?

Comment (mdahamiwal) [inline packages/office-ui-fabric-react/src/common/DelayRender.tsx]: I was not aware of that, will try reusing that. Thanks for the direction :)

Comment (mdahamiwal) [inline packages/office-ui-fabric-react/src/common/DelayRender.tsx]: @dzearing, I am using DelayedRender from utilities now. Its ready to have a relook. Thanks

**Comment by mdahamiwal** [comment]
@dzearing, can we merge this change? I have incorporated the review comments.

**Comment by mdahamiwal** [comment]
@micahgodbolt, can you review this?

**Pull request merged**